futures-rs-test-all/Cargo.toml

```diff
@@ -14,7 +14,7 @@ composability, and iterator-like interfaces.
 """
 
 [dependencies]
-log = "0.3"
+log = "= 0.3.6"
```

Comment:
I can change if you like, but I followed @nikomatsakis's example from the syntex .toml files.